### ISSUE TITLE
[Feature Request] tabs icons horizontal variant

### ISSUE THREAD

**Issue by iliyaZelenko** [vuetifyjs/vuetify#4461]
### Problem to solve
https://i.imgur.com/fQK1h9w.png

### Proposed solution
2 type prop: icons-and-text and icons-and-text-horizontal 
https://material.io/design/assets/1MxxR43Q_eSa9tFbIbBlPKVFZQ_r8FEKP/tabs-usage-010b.png

how at the moment to solve this problem?

**Comment by KaelWD** [comment]
`<v-icon left>`

The issue list of this repo is **exclusively** for bug reports and feature requests. For general questions, please join [the Discord chat room](https://community.vuetifyjs.com). You can also check [reddit](https://www.reddit.com/r/vuetifyjs/) or [stackoverflow](https://stackoverflow.com/questions/tagged/vuetify.js).

**Comment by iliyaZelenko** [comment]
in the tab it does not work, regardless of  icons-and-text prop
```
            <v-tab to="/" exact>
              <v-icon left>home</v-icon>
              Главаня
            </v-tab>
```

**Comment by jacekkarczmarczyk** [comment]
What doesn't work? https://codepen.io/anon/pen/NzJBgJ

**Comment by iliyaZelenko** [comment]
there must be an margin, "left" does not add it and I do not know how to add, I also tried class="mr-2"

**Comment by KaelWD** [comment]
🙄 Of course it only applies to buttons
![image](https://user-images.githubusercontent.com/16421948/42200849-82bf108a-7ed8-11e8-89ef-620895c9836d.png)

**Comment by jacekkarczmarczyk** [comment]
It was working long time ago 
https://github.com/vuetifyjs/vuetify/commit/f9b4a6f327125978edd00fc7f0348aa3783a3724#diff-2f2d1ff151f75bd54b921cc9ba6dcdd5L15

**Comment by johnleider** [comment]
This was resolved in f8de29dba0cca41ee6fd01c188cf1c8f1071b243

https://codepen.io/johnjleider/pen/pYEvQR
